packages/grafana-ui/src/components/ColorPicker/SeriesColorPickerPopover.tsx

```diff
@@ -22,18 +21,17 @@ export const SeriesColorPickerPopover = (props: SeriesColorPickerPopoverProps) =
           name: 'Y-Axis',
           tabComponent() {
             return (
-              <Switch
-                key="yaxisSwitch"
-                label="Use right y-axis"
-                className={styles.colorPickerAxisSwitch}
-                labelClass={styles.colorPickerAxisSwitchLabel}
-                checked={yaxis === 2}
-                onChange={() => {
-                  if (onToggleAxis) {
-                    onToggleAxis();
-                  }
-                }}
-              />
+              <InlineField labelWidth={20} label="Use right y-axis">
+                <InlineSwitch
+                  value={yaxis === 2}
+                  label="Use right y-axis"
+                  onChange={() => {
+                    if (onToggleAxis) {
```

Comment:
Don't think this condition is necessary since this code part already renders based on it.

Reply:
Great catch!